drivers/spi/spi_context.h

```diff
@@ -362,6 +365,9 @@ bool spi_context_tx_buf_on(struct spi_context *ctx)
 }
 
 static ALWAYS_INLINE
+/* Note: dfs is the number of bytes needed to store a data frame,
```

Comment:
Nit- can you use style like so:

```
/*
 * xxx
 */
```

to be consistent with other docstrings in this file? Ex: https://github.com/zephyrproject-rtos/zephyr/blob/b73c5578e3563f370d4fab0f7161a017020c114c/drivers/spi/spi_context.h#L412

Also, I'd move these comments above the `static ALWAYS_INLINE`

Reply:
I specifically did not do that on purpose and don't see why you think I would want to do your preferred style over mine but I don't care enough , so I updated the PR